src/content/en/fundamentals/design-and-ui/input/touch/index.md

```diff
@@ -457,7 +340,6 @@ of `pan-x | pan-y | manipulation` you prevent the default double-tap
 behavior.
 
 This allows you to implement a double-tap gesture yourself.
-In the case of IE10+, it also eliminates the 300ms click delay.
 
 Below is a list of the available parameters for *touch-action*.
```

Comment:
There are some more values now, but the only one I'd mention explicitly is `pinch-zoom`.  Sadly it's [not in the spec yet](https://github.com/w3c/pointerevents/issues/29), but IE, Edge, Chrome and Firefox will all support it.  I.e. `touch-action: pan-y pinch-zoom` allows vertical scrolling without also disabling pinch-zoom.  This should arguably be recommended over bare `pan-y` for accessibility reasons.